<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplashForm</class>
 <widget class="QWidget" name="SplashForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QProgressBar{
        border: none;
        color: white;
        text-align: center;
        background: rgb(68, 69, 73);
}
QProgressBar::chunk {
        border: none;
        background: rgb(0, 160, 230);
}

QWidget#desktop {
        background: rgb(68, 69, 73);
}
QWidget{
        font-size: 15px;
        font-family: &quot;arial&quot;;

}

QLabel#label {
        font: bold;
        font-family: &quot;arial&quot;;
        color: rgb(255, 0, 0);
        font-size: 20px;
}

</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="lab_info">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="WidgetBase" name="wid_icon" native="true">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>100</y>
     <width>150</width>
     <height>200</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetBase</class>
   <extends>QWidget</extends>
   <header>widgetbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
